pkg/cli/cmd/credential/show/objectformats.go

```diff
@@ -19,32 +19,70 @@ package show
 import (
 	"strings"
 
+	"github.com/radius-project/radius/pkg/cli/credential"
 	"github.com/radius-project/radius/pkg/cli/output"
 )
 
+const (
+	AzureServicePrincipalCredentialKind = "ServicePrincipal"
+	AzureWorkloadIdentityCredentialKind = "WorkloadIdentity"
```

Comment:
These are defined in multiple files, can they be consolidated?

Reply:
unified to be the ucp datamodel value